brave-instrumentation/aws-java-sdk-core/src/test/java/brave/instrumentation/aws/ITTracingRequestHandler.java

```diff
@@ -13,49 +13,36 @@
  */
 package brave.instrumentation.aws;
 
-import brave.ScopedSpan;
 import brave.SpanCustomizer;
-import brave.Tracer;
 import brave.http.HttpAdapter;
 import brave.http.HttpClientParser;
-import brave.http.HttpTracing;
-import brave.internal.HexCodec;
-import brave.sampler.Sampler;
-import brave.test.http.ITHttpAsyncClient;
+import brave.http.HttpResponseParser;
+import brave.http.HttpTags;
+import brave.test.http.ITHttpClient;
 import com.amazonaws.ClientConfiguration;
 import com.amazonaws.auth.AWSStaticCredentialsProvider;
 import com.amazonaws.auth.BasicAWSCredentials;
 import com.amazonaws.client.builder.AwsClientBuilder;
 import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
 import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClientBuilder;
-import java.util.ArrayList;
-import java.util.Arrays;
+import java.io.IOException;
 import java.util.Collections;
-import java.util.List;
-import java.util.concurrent.TimeUnit;
-import java.util.stream.Collectors;
 import okhttp3.mockwebserver.MockResponse;
-import okhttp3.mockwebserver.RecordedRequest;
-import okhttp3.mockwebserver.SocketPolicy;
 import org.junit.Test;
 import zipkin2.Span;
 
-import static java.util.Arrays.asList;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class ITTracingRequestHandler extends ITHttpAsyncClient<AmazonDynamoDB> {
-  @Override protected void getAsync(AmazonDynamoDB dynamoDB, String s) {
-    dynamoDB.getItem(s, Collections.emptyMap());
-  }
-
+public class ITTracingRequestHandler extends ITHttpClient<AmazonDynamoDB> {
   @Override protected AmazonDynamoDB newClient(int i) {
     ClientConfiguration clientConfiguration = new ClientConfiguration();
-    clientConfiguration.setMaxErrorRetry(2);
+    clientConfiguration.setMaxErrorRetry(0);
     clientConfiguration.setRequestTimeout(1000);
 
     return AmazonDynamoDBAsyncClientBuilder.standard()
```

Comment:
Recommend `AmazonDynamoDBClientBuilder` to make clear it's not an accident we're testing sync only

Reply:
good idea